Code/GraphMol/testPicklerGlobalSettings.cpp

```diff
@@ -0,0 +1,239 @@
+// $Id$
```

Comment:
This header should be cleaned up: the Id thing is an SVN dropping and the copyright is wrong.